src/playground/sample_data_tooling/sample_data_tests/sample_data_commons_tests/test-files/example_config_ndjson.ndjson

```diff
@@ -0,0 +1 @@
+{"key": "value"}
```

Comment:
You should have a sample of ndjson. your value here is the exactly same as json file. please give a good sample for ndjson, it should not just one line vs multi-lines. Thanks

Reply:
Done, `.ndjson` file has multiple lines with different key-values.